lib/ui/pages/views/facilities_view/facilities_view_model.dart

```diff
@@ -1,8 +1,35 @@
+import 'package:Covid19_PH/model/hospital_summary.dart';
+import 'package:Covid19_PH/services/hospital_database.dart';
 import 'package:stacked/stacked.dart';
 
-class FacilitiesViewModel extends BaseViewModel {
-  initialise() {
-    print('Facilities View Model Initialized');
-    notifyListeners();
+class FacilitiesViewModel extends FutureViewModel<HospitalSummary> {
+  final _database = HospitalDatabase.instance;
+
+  Future<HospitalSummary> fetchFacilitiesData() async =>
+      await _database.fetchHospitalRecordsSummary();
+
+  String headerTitle = 'Summary of Philippine Hospital Facilities Number';
```

Comment:
No need for this, 'cause the hospital name will be included in json response.
Will remove this in the next, PR.